mllib/src/main/scala/org/apache/spark/mllib/evaluation/RankingMetrics.scala

```diff
@@ -156,6 +156,32 @@ class RankingMetrics[T: ClassTag](predictionAndLabels: RDD[(Array[T], Array[T])]
     }.mean()
   }
 
+  /**
+   * Returns the mean reciprocal rank (MRR) of all the queries.
```

Comment:
It would be good to expand the doc string here with more detail about what is MRR and how it is computed (see the `ndcg` doc string above).